index.html

```diff
@@ -13,27 +21,365 @@
         <div class="logo">
             <img src="./src/logo.svg" alt="logo">
         </div>
-        <div class="navbar ">
+        <div class="navbar normal-text">
             <ul>
                 <li><a href="#">Home </a></li>
                 <li><a href="#">Deals</a></li>
                 <li><a href="#">New Arrivals</a></li>
                 <li><a href="#">Packages</a></li>
                 <li><a href="#">Sing in</a></li>
             </ul>
-            <a href="#" class="button">Sign Up</a>
+            <a href="#" class="button ">Sign Up</a>
         </div>
     </div>
     <div class="ultimate-sale wrapper">
         <img src="./src/ultimate-sale.png" alt="ultimate-sale">
     </div>
-    <div class="brands-logo wrapper">
+    <div class="brands-logo wrapper ">
         <img src="./src/chanel.png" alt="chanel" width="196px" height="33px">
         <img src="./src/louis-vuitton.png" alt="louis-vuitton" width="196px" height="25px">
         <img src="./src/prada.png" alt="prada" width="196px" height="32px">
         <img src="./src/calvin-klein.png" alt="calvin-klein" width="196px" height="33px">
         <img src="./src/denim.png" alt="denim" width="184px" height="27px">
```

Comment:
```suggestion
        <img src="./src/chanel.png" alt="chanel" width="196" height="33">
        <img src="./src/louis-vuitton.png" alt="louis-vuitton" width="196" height="25">
        <img src="./src/prada.png" alt="prada" width="196" height="32">
        <img src="./src/calvin-klein.png" alt="calvin-klein" width="196" height="33">
        <img src="./src/denim.png" alt="denim" width="184" height="27">
```
Recheck all values like this